### ISSUE TITLE
Cache Fails when Multiple Accounts are Cached on iOS

### ISSUE THREAD

**Issue by hncleary** [recognizegroup/capacitor-plugin-msauth#16]
I encountered an issue that occurs when multiple apps using this plugin are on the same device.

```
Unable to query current account: Error Domain=MSALErrorDomain Code=-50000 "(null)" UserInfo={MSALErrorDescriptionKey=Multiple accounts found in cache., MSALInternalErrorCodeKey=-42102}
```

I am currently developing a set of branded applications. Although their functionality is similar - each instance will be its own entity with a unique bundle identifier.

It seems that logging into these separate applications (even with the same account) on a device will leave multiple accounts in the cache. Any difference in the msal authority will create a new cached account. 

Currently the plugin has no method for handling multiple accounts. This breaks the cache and causes users to have to input their login information at the start of every session. Addiontally, deleting an app from the device does not delete its associated cache.

**Comment by hncleary** [comment]
Would like to specify that after testing thoroughly on Android devices nothing simliar to this issue was ecountered.

**Comment by bartwesselink** [comment]
Closed because #17 was merged.
